.github/workflows/codeql.yaml

```diff
@@ -37,18 +37,22 @@ jobs:
         restore-keys: |
           ${{ github.job }}-${{ runner.os }}-go-
 
-    - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
-      with:
-        languages: go
-
-    # Install IBM MQ redis client required by IBM MQ source and target adapters
+    # Install IBM MQ client required by IBM MQ source and target adapters
     - name: Install IBM MQ client
       run: |
         curl https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqdev/redist/9.2.3.0-IBM-MQC-Redist-LinuxX64.tar.gz -o mq.tar.gz
         mkdir -p /opt/mqm
         tar -C /opt/mqm -xzf mq.tar.gz
 
+    # Install C libraries required by the XSLT transformation adapter
+    - name: Install C libraries for XSLT transformation
+      run: apt-get install -y --no-install-recommends libxml2-dev libxslt1-dev liblzma-dev zlib1g-dev
+
+    - name: Initialize CodeQL
```

Comment:
Surprised by the ordering. Would've thought the missing MQ components would have triggered a failure as well.

Reply:
Apparently the "initialize" step only prepares the environment somehow, it doesn't yet build the code so that's why it works. Still, I think it's more logical to keep CodeQL related steps together.